### PR TITLE
fix(FEC-7235): ads container accessible play/pause area.

### PR DESCRIPTION
**Pull request by dvh91**
### Description of the Changes

Listening to ads-container click event in order to enable play/pause in mobile.

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/overlay-play/overlay-play.js [hunk 1]
```diff
@@ -30,14 +31,38 @@ const mapStateToProps = state => ({
...
+   * @returns {void}
+   * @memberof OverlayPlay
+   */
+  componentDidUpdate() {
+    if (!this.adsContainerClickHandlerInitialized && this.props.metadataLoaded) {
+      let player = document.getElementById(this.config.targetId);
+      let adsContainer = [];
+
+      if (player) {
+        adsContainer = player.querySelectorAll('*[id^="ads-container_"]');
```
Comment (OrenMe) [inline src/components/overlay-play/overlay-play.js]: I think that we should move this actually to owner of this overlay - in the ads plugin itself

Comment (dvh91) [reply src/components/overlay-play/overlay-play.js]: @OrenMe - If this will come with configuration of if to have it or not, it could be a good idea.

Comment (OrenMe) [reply src/components/overlay-play/overlay-play.js]: @dan-ziv WDYT?

**Comment by dan-ziv** [comment]
closing this PR
will be done inside the ads plugin

**Pull request closed without merging**